sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -2596,6 +2596,10 @@ def __GetBodiesFromQueryResult(result):
                 return __GetBodiesFromQueryResult(results)
 
         result, self.last_response_headers = self.__Post(path, request_params, query, req_headers, **kwargs)
+        if http_constants.HttpHeaders.IndexUtilization in self.last_response_headers:
+            INDEX_METRICS_HEADER = http_constants.HttpHeaders.IndexUtilization
+            index_metrics_raw = self.last_response_headers[INDEX_METRICS_HEADER]
```

Comment:
kind of a nitpick since we know the headers aren't so large of a dict that performance would be very different but

```suggestion
        if self.last_response_headers.get(http_constants.HttpHeaders.IndexUtilization) is not None:
            index_metrics_raw = self.last_response_headers[http_constants.HttpHeaders.IndexUtilization]
```